README.md

```diff
@@ -32,6 +31,17 @@ npm i
 
 ### Local SolidityX Compiler
 
+There are two methods of installing the SolidityX compiler for use with Hardhat:
+
+- Install the SolidityX compiler via [`quai-hardhat-plugin`](https://www.npmjs.com/package/quai-hardhat-plugin) (**Recommended**)
+- Install and build the SolidityX compiler from source
+
+#### Installing via Plugin
+
+If you've installed `quai-hardhat-plugin` already, the SolidityX compiler will be installed automatically when you run `npx hardhat compile` for MacOS and Linux users. Windows is not currently supported by the plugin.
+
+#### Installing from Source
+
 Visit the [SolidityX Repository](https://github.com/dominant-strategies/SolidityX) for instructions on how to clone and build the SolidityX compiler for your specific operating system.
 
 Once you've built the SolidityX compiler, you'll need to add path to your `solc` binary into the `customCompilerPath` variable in the `hardhat.config.js` file. The file already includes common paths for MacOS and Linux as comments.
```

Comment:
You still need quai-hardhat-plugin even if you install from source

Reply:
Fixed with [aa071e9](https://github.com/dominant-strategies/hardhat-example/pull/27/commits/aa071e92f813592d286f3c44ed45f2949aa65926)